packages/fiber/src/react/reconciler.ts

```diff
@@ -0,0 +1,216 @@
+import {
+  appendChild,
+  createInstance,
+  insertBefore,
+  Instance,
+  InstanceProps,
+  LocalState,
+  prepare,
+  removeChild,
+  Root,
+} from '../core/renderer'
+import { attach, detach, diffProps, DiffSet, invalidateInstance, is, applyProps } from '../core/utils'
+import Reconciler from 'react-reconciler'
+import { UseBoundStore } from 'zustand'
+import { RootState } from '../core/store'
+
```

Comment:
The reconciler was the React-specific part of the createRenderer, splitting it into two parts, where the core exposes a renderer and the ReactReconciler operates using the renderer.